src/components/utils/popover.ts

```diff
@@ -7,9 +7,9 @@ import { isMobileScreen, keyCodes, cacheable } from '../utils';
 import ScrollLocker from './scroll-locker';
 
 /**
- * Describe parameters for rendering the single item of Popover
+ * Describe parameters for rendering a single popover item
  */
-export interface PopoverItem {
+export interface PopoverItemBase {
```

Comment:
lets move all the types to the `/types/configs/popover.d.ts` (and export them from types/index.d.ts)

Reply:
done